build.sbt

```diff
@@ -4,15 +4,28 @@ version := "0.1"
 
 scalaVersion := "2.11.12"
 
+
 sparkVersion := "2.2.2"
 sparkComponents ++= Seq("sql")
 
-libraryDependencies += "org.apache.spark" %% "spark-streaming" % "2.2.2" % "provided"
+libraryDependencies += "org.apache.spark" %% "spark-streaming" % "2.2.2"
+
+libraryDependencies += "org.apache.spark" %% "spark-sql" % "2.2.2"
 
 // https://mvnrepository.com/artifact/org.apache.spark/spark-streaming-twitter
-libraryDependencies += "org.apache.spark" %% "spark-streaming-twitter" % "1.6.3"
+libraryDependencies += "org.apache.bahir" %% "spark-streaming-twitter" % "2.2.1"
+
+
+
```

Comment:
I believe the empty lines are redundant.
Please keep attention to formatting. 
The goal is to keep codebase clean and beautiful :)

Reply:
Fixed..